selfdrive/ui/qt/widgets/ssh_keys.cc

```diff
@@ -26,7 +26,7 @@ SshControl::SshControl() : AbstractControl("SSH Keys", "Warning: This grants SSH
 
   QObject::connect(&btn, &QPushButton::released, [=]() {
     if (btn.text() == "ADD") {
-      QString username = InputDialog::getText("Enter your GitHub username");
+      QString username = InputDialog::getText("Enter your GitHub username", 39, this);
```

Comment:
min length added by mistake?

Reply:
github limits the username to 39 characters.

Reply:
This is a minimum length (e.g. for wifi passwords).

Reply:
....my bad